src/app/course-item/course-item.component.spec.ts

```diff
@@ -1,6 +1,8 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { CourseItemComponent } from './course-item.component';
+import {Component, NO_ERRORS_SCHEMA} from "@angular/core";
```

Comment:
Please check code formatting

Reply:
Thanks, it is just default settings for autocomlete in Webstrom :)